src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java

```diff
@@ -27,7 +27,7 @@
   @FindBy(css = ".local-wikia-search a")
   private WebElement searchSuggestion;
 
-  @FindBy(xpath = "//ul[contains(@class, \"local-nav-menu\")]/li/*")
```

Comment:
http://stackoverflow.com/questions/16788310/what-is-the-difference-between-css-selector-xpath-which-is-betteraccording-t